sites/partners/__tests__/pages/listings/lottery.test.tsx

```diff
@@ -114,12 +122,15 @@ describe("lottery", () => {
       })
     )
 
-    const { getAllByText, findByText } = render(<Lottery listing={listing} />)
+    const { getAllByText, getByText, findByText } = render(<Lottery listing={closedListing} />)
 
     const header = await findByText("Partners Portal")
     expect(header).toBeInTheDocument()
 
-    expect(getAllByText(listing.name).length).toBeGreaterThan(0)
+    expect(getAllByText(closedListing.name).length).toBeGreaterThan(0)
+    expect(
+      getByText("Lottery data for this listing will expire on", { exact: false })
```

Comment:
Can we have one test that tests partner + expiry that mocks the expired status? I wouldn't expect this string to exist here unless the status is explicitly set